roles/libvirt_manager/tasks/create_cloud_init_iso.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Generate cloud-init iso
+  become: false
+  vars:
+    _images_dir: "{{ [ cifmw_libvirt_manager_basedir, 'workload', ] | ansible.builtin.path_join }}"
+  block:
+    - name: "Acquire authorized_keys from hypervisor"
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir ~ '/.ssh/authorized_keys' }}"
+      register: _authorized_keys_b64
+
+    - name: "Generate instance random UUID"
+      ansible.builtin.set_fact:
+        _instance_uuid: "{{ 99999999 | random | to_uuid | lower }}"
+
+    - name: "Render vm type based user-data"
+      vars:
+        _authorized_keys: "{{ _authorized_keys_b64.content | b64decode }}"
+        _static_ip_script: >-
+          {{
+            (
+              cifmw_libvirt_manager_ip_script_basedir,
+              'static-ip-' + item.key + '.sh'
+            ) | ansible.builtin.path_join
+          }}
+      ansible.builtin.set_fact:
+        _userdata: "{{ lookup('ansible.builtin.template', 'user-data.yml.j2') }}"
```

Comment:
Do we need this template? Can we not just build up cifmw_config_drive_userdata? similar to - https://github.com/openstack-k8s-operators/ci-framework/pull/1813/files#diff-0b33aeeecf16501c026ef2b08cfd8eb03d767fb837e927e630ff6c8dd5ebb356R96-R130